frontend/packages/employee-frontend/src/types/absence.ts

```diff
@@ -53,11 +53,18 @@ export const billableCareTypes: CareType[] = ['PRESCHOOL_DAYCARE', 'DAYCARE']
 export function formatCareType(
   careType: CareType,
   placementType: PlacementType,
+  dateOfBirth: LocalDate,
   i18n: Translations
 ) {
   const isPreparatory =
     placementType === 'PREPARATORY' || placementType === 'PREPARATORY_DAYCARE'
 
+  const hasFree5YearsOldDaycare =
+    dateOfBirth.year === LocalDate.today().year - 5
```

Comment:
This is not how it's determined. Children that turn 5 in year xxxx get the discount during the "season" of 1.8.xxxx - 31.7.(xxxx + 1), so basically for the year before preschool.

Reply:
Oh.. argh